_projects/2024/100535296/100535296.Rmd

```diff
@@ -0,0 +1,534 @@
+         (Count>=400) ~ "400 or higher"))
+
+data_counts$Age <- as.numeric (data_counts$Age)
+```
+
+Next, we renamed the sports to match the terminology used in the original graph, as "Rhythmic Gymnastics" and "Synchronized Swimming" were written differently. Following this, we ordered the sports alphabetically, from A to Z, in accordance with the structure of the original graph.
+
+```{r}
+#Rythmic gymnastics into Rythimc Gym.
+
+data_counts <- data_counts |> 
+  mutate(Sport = ifelse(Sport == "Rhythmic Gymnastics", "Rhythmic Gym.", Sport))
+
+# Synchronized Swimming into Sync. Swimming
+
+data_counts <- data_counts |> 
+  mutate(Sport = ifelse(Sport == "Synchronized Swimming", "Sync. Swimming", Sport))
+
+#Ordering sports alphabetically
+
+data_counts <- data_counts |> 
+  arrange(Sport) |> 
+  mutate(Sport = factor(Sport, levels = rev(sort(unique(Sport)))))
+```
+
+Finally, we create a new table using summarise to calculate the minimum and maximum ages for each sport in order to be able to add this information more easily to the graph later. Afterward, we merged this table with the original one using left_join.
+
+```{r}
+age_extremes <- data_counts |> 
+  group_by(Sport) |> 
+  summarise(min_age = min(Age, na.rm = TRUE), 
+            max_age = max(Age, na.rm = TRUE)) |> 
+  ungroup() 
+
+data_counts <- data_counts |> 
+  left_join(age_extremes, by = "Sport")
+
+```
+
+## BUILDING THE GRAPH
+
+Once the dataset was cleaned and prepared, we moved on to the process of replicating the graph. To ensure the final output closely resembled the original, we began by downloading the necessary resources, particularly the fonts used in the visualization.
+
+### FONTS
+
+In order to replicate the font styles of the original graph, we focused on matching all the fonts displayed, which we identified through The Washington Post's design system page. This resource provides details on the fonts they use, including their size, weight, and height specifications for articles: https://build.washingtonpost.com/foundations/typography. 
+
+We were able to recover all the required fonts, except for the title font, Postoni, which is a paid font. As an alternative, we chose Georgia, a font also used by The Washington Post for article text.
+
+To implement these fonts in the graph, we used the showtext package, which allows for text rendering using fonts downloaded from Google Fonts, such as Libre Franklin, the font selected for annotations. Additionally, we employed the extrafont package, which simplifies the registration and use of external fonts installed on the operating system. We used Georgia for the title, subtitle, and caption, while Arial was chosen for the numbers indicating the maximum and minimum ages of athletes in each sport.
+
+```{r}
+
+font_add_google("Libre Franklin", "franklin")
+showtext_auto() 
+
+font_add(family = "Georgia", regular = "C:/Windows/Fonts/georgia.ttf")
+font_add(family = "Arial", regular = "C:/Windows/Fonts/arial.ttf")
```

Comment:
This should be avoided. These files exist in your computer, but not in e.g. mine. I prefer if you select some font from Google Fonts that is close but not exact rather than this.